lib/active_merchant/billing/gateways/elavon.rb

```diff
@@ -74,6 +75,7 @@ def capture(money, authorization, options = {})
           add_invoice(form, options)
           add_creditcard(form, options[:credit_card])
           add_currency(form, money, options)
+          add_address(form, options)
```

Comment:
This additional line to send the customer's address in the `capture` method was done to get the `test_authorize_and_capture` remote test to pass. The test was previously failing due to the `avs_address` field not being sent to the gateway as part of the `capture` request.